docs/guides/cli/17_configure-permission-service.mdx

```diff
@@ -77,3 +79,61 @@ ory patch permission-config <project-id> \
 Read [this document](./config-with-cli) to learn more about adjusting project configuration with the Ory CLI.
 
 :::
+
+## Manage relation tuples
+
+You can manage the [relation tuples](../../keto/concepts/relation-tuples) directly from the Ory CLI as well. Here's how:
+
+### Setup
+
+```bash
+export PROJECT=<your_project_id>
+
+## Authenticate first
+ory auth
+# (sign in with Ory Cloud account)
+```
+
+### Create some relation tuples
+
+[Detailed documentation for `ory create relation-tuples`](../../cli/ory-create-relation-tuples)
+
+```bash
+echo "groups:devs#member@tom" \
+  | ory parse relation-tuples --project=$PROJECT --format=json - \
+  | ory create relation-tuples --project=$PROJECT -
+# NAMESPACE	OBJECT	RELATION NAME	SUBJECT
+# groups		devs	member		tom
+```
+
+### List the tuple(s)
+
+[Detailed documentation for `ory list relation-tuples`](../../cli/ory-list-relation-tuples)
+
+```bash
+ory list relation-tuples --project=$PROJECT
+# NAMESPACE	OBJECT	RELATION NAME	SUBJECT
+# groups		devs	member		tom
+#
+# NEXT PAGE TOKEN
+# IS LAST PAGE	true
+```
+
+### Delete the tuple(s)
+
+[Detailed documentation for `ory delete relation-tuples`](../../cli/ory-delete-relation-tuples)
+
+```bash
+ory delete relation-tuples --project=$PROJECT --all
+# WARNING: This operation is not reversible. Please use the `--force` flag to proceed.
+# These tuples would be deleted:
+#
+# NAMESPACE	OBJECT	RELATION NAME	SUBJECT
+# groups		devs	member		tom
+#
+# NEXT PAGE TOKEN
+# IS LAST PAGE	true
+
+## Delete the tuple(s) for real
+ory delete relation-tuples --project=$PROJECT --all --force
+```
```

Comment:
The documented behavior is unnecessarily surprising to me:

- `ory delete` doesn't delete things but only prints the things to be deleted
- `ory delete --force` deletes them for real

I'd like to propose this behavior instead (which I have seen in the wild):

- `ory delete --dry-run` shows what it would delete, but doesn't delete things
- `ory delete` deletes things
- `ory delete --force` deletes things that are normally wouldn't get deleted, like tuples that are "locked" or still used somewhere. `--force` is kind of like `sudo`. One should never use it unless having a very good reason, knowing what one is doing, and thinking before typing.

Alternative behavior:

- `ory delete` asks for manual confirmation about each entry to be deleted
- `ory delete --yes` skips the confirmation and deletes right away